Detectors/Raw/src/RawFileReader.cxx

```diff
@@ -24,7 +24,8 @@ using namespace o2::raw;
 //____________________________________________
 void RawFileReader::LinkBlock::print() const
 {
-  printf("file:%3d offs:%10zu size:%8d newSP:%d newTF:%d newHB:%d\n", fileID, offset, size, startSP, startTF, startHB);
+  printf("file:%3d offs:%10zu size:%8d newSP:%d newTF:%d newHB:%d endHB:%d | Orbit %u\n",
+         fileID, offset, size, startSP, startTF, startHB, endHB, orbit);
```

Comment:
Think the Framework/Logger should be used, https://github.com/AliceO2Group/AliceO2/tree/dev/Framework/Logger, e.g. the LOGF macro.